Makefile

```diff
@@ -3,57 +3,56 @@
 # Run Postgres
 .PHONY: db
 db:
-  docker start codesearch-db || docker run --name codesearch-db \
-    -e POSTGRES_DB=sourcesdb \
-    -e POSTGRES_USER=postgres \
-    -e POSTGRES_PASSWORD=postgres \
-    -p 5432:5432 -d postgres -N 1000
+	docker start codesearch-db || docker run --name codesearch-db \
+		-e POSTGRES_DB=sourcesdb \
+		-e POSTGRES_USER=postgres \
+		-e POSTGRES_PASSWORD=postgres \
+		-p 5432:5432 -d postgres -N 1000
 
 # Connect to Postgres
 .PHONY: psql
 psql:
-  docker run -it --rm --link codesearch-db:postgres \
-    -e PGPASSWORD=postgres \
-    postgres psql -h postgres -U postgres sourcesdb
+	docker run -it --rm --link codesearch-db:postgres \
+		-e PGPASSWORD=postgres \
+		postgres psql -h postgres -U postgres sourcesdb
 
 # Create tables
 .PHONY: tables
 tables:
-  java -jar codesearch-core.jar -i
+	java -jar codesearch-core.jar -i
 
 # Destroy Postgres
 .PHONY: db-kill
 db-kill:
-  docker stop codesearch-db
-  docker rm codesearch-db
+	docker stop codesearch-db
+	docker rm codesearch-db
 
 # Build the project
-.PHONY: build
 build:
-  sbt core/assembly web-server/assembly
+	sbt core/assembly web-server/assembly
 
 # Download packages. Acceptable values: {haskell, rust, ruby, javascript}
 download-%:
-  java -jar codesearch-core.jar -d -u -l "$*"
+	java -jar codesearch-core.jar -d -u -l "$*"
 
 # Index packages. Acceptable values: same as for download-%
 index-%:
-  java -jar codesearch-core.jar -b -l "$*"
+	java -jar codesearch-core.jar -b -l "$*"
 
 # Run the server
 .PHONY: serve
 serve:
-  java -jar codesearch-server.jar
+	java -jar codesearch-server.jar
 
 # Build a Docker image (the project must be built already)
-docker-%:
-  docker build \
-    -f "docker/$*/Dockerfile" \
-    -t "quay.io/aelve/codesearch-$*:local" .
-  docker tag \
-    "quay.io/aelve/codesearch-$*:local" \
-    "quay.io/aelve/codesearch-$*:latest"
+build-docker-%:
+	docker build \
+		-f "docker/$*/Dockerfile" \
+		-t "quay.io/aelve/codesearch-$*:local" .
+	docker tag \
+		"quay.io/aelve/codesearch-$*:local" \
+		"quay.io/aelve/codesearch-$*:latest"
 
 # Push a Docker image to Quay
-docker-push-%:
-  docker push "quay.io/aelve/codesearch-$*:latest"
+push-docker-%:
```

Comment:
Don't forget to update readmes and the wiki (if needed) once this is merged. I think those commands were mentioned in the readmes somewhere, but I'm not sure.

Reply:
Done.